src/test/java/com/salesforce/storm/spout/dynamic/VirtualSpoutTest.java

```diff
@@ -271,14 +265,12 @@ public void testCallingOpenTwiceThrowsException() {
             eq(null)
         );
 
-        // Set expected exception
-        try {
-            expectedExceptionCallingOpenTwiceThrowsException.expect(IllegalStateException.class);
-            virtualSpout.open();
-        } finally {
-            // Ensure that we call close.
-            virtualSpout.close();
-        }
+        Assertions.assertThrows(IllegalStateException.class, () ->
```

Comment:
these also worry me the same as the other ones.  You may want to verify that if  virtualSpout.open() does NOT throw an illegalstateexception, or some other class, that vspout.close() below gets called.